guidelines/groups/process-and-task-completion/avoid-deception.md

```diff
@@ -1,10 +1,17 @@
 ---
 children:
-  - deceptive-controls
-  - exploitive-behaviors
-  - misinformation
-  - preselections
-  - redirection
+  - changes-in-agreement
+  - no-misleading-wording
+  - no-artificial-pressure
+  - no-misinformation
+  - no-hidden-preselections
+  - no-sneaking
+  - no-interference
```

Comment:
`groups/process-and-task-completion/avoid-deception/no-interference.md` does not exist